zebrad/src/components/sync/downloads.rs

```diff
@@ -169,6 +182,16 @@ where
                     rsp = block_req => rsp?,
                 };
 
+                // Check if the block is already in the state.
+                // BUG: check if the hash is in any chain (#862).
+                // Depth only checks the main chain.
```

Comment:
We are repeating this bug here. I dont think this is a problem in this PR because the check in the `BlockVerifier` was also repeating the bug even if it had no `BUG:` comment.

Reply:
We'll fix this bug when we fix #862.